.github/workflows/build_and_test.yml

```diff
@@ -601,6 +607,8 @@ jobs:
       run: ./dev/lint-java
     - name: Python linter
       run: PYTHON_EXECUTABLE=python3.9 ./dev/lint-python
+    - name: Python code generation check
+      run: PATH=$PATH:$HOME/buf/bin PYTHON_EXECUTABLE=python3.9 ./dev/check-codegen-python.py
```

Comment:
Let's skip this test if the script doesn't exist. This workflow runs in branch-3.2 and 3.3 too (see https://github.com/apache/spark/actions/runs/3263093569/jobs/5361307965)

Reply:
got it, let me fix it